src/clusterfuzz/_internal/build_management/build_manager.py

```diff
@@ -463,8 +525,7 @@ def _unpack_build(self, base_build_dir, build_dir, build_url):
               'Cleared all data directories to free up space, exiting.')
 
         # Unpack the local build archive.
-        logs.info(
-            f'Unpacking build archive {build_local_archive} to {build_dir}.')
+        logs.info(f'Unpacking build archive {build_url} to {build_dir}.')
```

Comment:
It seems we're no longer logging the path to the zip file we downloaded (in the event we did download the full zip) anywhere. Can you add that to the log on line 432?